pom.xml

```diff
@@ -59,7 +59,7 @@
         <revision>2.35</revision>
         <gitHubRepo>jenkinsci/pipeline-stage-view-plugin</gitHubRepo>
         <bom>2.361.x</bom>
-        <jenkins.version>2.361.4</jenkins.version>
+        <jenkins.version>2.462.1</jenkins.version>
```

Comment:
Mismatch. See https://github.com/jenkinsci/archetypes/pull/737. Anyway this does not seem related to the rest of the PR and would better be reverted (there could be a separate PR to refresh dependencies).

Reply:
Will revert tomorrow. Thank you

Reply:
Done!

Reply:
Done in the sense that the changes have been reverted back. Thanks